src/main/java/com/harrison/controllers/EventController.java

```diff
@@ -0,0 +1,114 @@
+package com.harrison.controllers;
+
+import java.math.BigInteger;
+import java.util.Arrays;
+import java.util.Comparator;
+import java.util.HashMap;
+import java.util.List;
+import java.util.Map;
+import java.util.stream.Collectors;
+
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.web.bind.annotation.PathVariable;
+import org.springframework.web.bind.annotation.RequestBody;
+import org.springframework.web.bind.annotation.RequestMapping;
+import org.springframework.web.bind.annotation.RequestMethod;
+import org.springframework.web.bind.annotation.RestController;
+
+import com.harrison.domain.EventComponent;
+import com.harrison.domain.EventUnitSource;
+import com.harrison.repository.EventComponentRepository;
+import com.harrison.repository.EventUnitSourceRepository;
+import com.harrison.views.Event;
+
+@RestController
+@RequestMapping(value = "/api/event")
+public class EventController {
+	
+	private static final String ALPHA = "ALPHA";
+
+	private static final String CHARLIE = "CHARLIE";
+
+	@Autowired
+	private EventComponentRepository eventComponentRepository;
+	
+	@Autowired
+	private EventUnitSourceRepository eventUnitSourceRepository;
+	
+	@RequestMapping(value = "eventcomponents", method = RequestMethod.GET)
+	public List<EventComponent> getEvents() {
+	    String[] data = new String[6];
+	    Arrays.sort(data, new Comparator<String>() {
```

Comment:
![MAJOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-major.png 'Severity: MAJOR') Make this anonymous inner class a lambda (sonar.java.source not set. Assuming 8 or greater.) [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](http://ec2-54-187-222-128.us-west-2.compute.amazonaws.com:9000/sonar/coding_rules#rule_key=squid%3AS1604)